airflow/models/baseoperator.py

```diff
@@ -1693,6 +1698,16 @@ def inherits_from_empty_operator(self):
         # of its subclasses (which don't inherit from anything but BaseOperator).
         return getattr(self, "_is_empty", False)
 
+    @property
+    def start_trigger(self) -> BaseTrigger | None:
+        """Trigger when deferring task."""
+        return self._start_trigger
+
+    @property
+    def next_method(self) -> str | None:
+        """Method to execute after finish deferring."""
+        return self._next_method
```

Comment:
Why do we need to do this attribute-property dance? I wonder if these can just be public attributes instead.

Reply:
The original idea was to prevent the DAG author from assigning this value. e.g.,

```python
op = SomeOperator(task_id="task_id")
op.start_trigger = ...
op.next_method

# They can still do op._start_trigger though
```

Not sure whether it's a valid concern

Reply:
I don’t think this _technically_ breaks anything though? And we already have a lot of such attributes that don’t work with reassignments. I think it should be fine as long as the attribute is not documented as settable.

Reply:
Sounds good. 👍  Let me change it back to public attribute instead.

Reply:
Just updated it 🚀